libs/@local/hash-backend-utils/src/pg-tables.ts

```diff
@@ -2,74 +2,106 @@
  * This module contains type definitions for tables in the Postgres database. Column
  * names are converted from snake_case to camelCase for consistency.
```

Comment:
This seems outdated

Reply:
Yes, 36bf50b179768275c97d5547d620d912ba8d02e9